docs/docs/guides/proxy.mdx

```diff
@@ -171,28 +170,30 @@ Cookie:
 }
 ```
 
-## Ory Proxy for Local Development
+## Local Development
```

Comment:
Keeping names explicit in the headlines is good for SEO. If someone searches "ory proxy" on google, it's more likely that they'll end up on this page!

Reply:
Makes sense, reverted it!